grails-app/i18n/messages.properties

```diff
@@ -3678,6 +3678,7 @@ react.partialReceiving.error.invalidDate.label=This date is invalid. Please ente
 react.partialReceiving.error.expiryWithoutLot.label=Items with an expiry date must also have a lot number.
 react.partialReceiving.error.lotAndExpiryControl.label=Both lot number and expiry date are required for this item.
 react.partialReceiving.confirmReceive.notSupportingPartialReceiving.message=You have not entered a value in the receiving quantity for ${count} line/lines on this shipment. These lines will be received as zero. You will not be able to go back and receive them later. Do you want to continue?
+react.partialReceiving.mismatchingQuantityShipped.label=The total edited quantity does not match the original quantity shipped.
```

Comment:
I think we should follow one pattern when naming those labels. I saw that we are using `.error.` in the labels.
So maybe we can call it like: `react.partialReceiving.error.mismatchingQuantityShipped.label`

Reply:
yeah I agree, I never feel comfortable adding a new label because I always wonder if any pattern is applicable there.